### ISSUE TITLE
[Feature Request] Prompt users whether to install optional dependencies

### ISSUE THREAD

**Issue by mooziii** [jakobkmar/pacmc#55]
title says it 
Example:

```
Do you want to install any optional dependencies?
1) modrinth/modmenu
2) modrinth/clothconfig
3) modrinth/fabric-api

List the dependencies that you want to install (provide the numbers): 2,3
```
^^^
this would install clothconfig and fabric api but not modmenu

**Comment by jakobkmar** [comment]
Do you think this prompt should be default behaviour or only appear with a special flag?

**Comment by mooziii** [comment]
Should be default

**Comment by sparrowofsong** [comment]
I agree; I'm using quilt, which replaces fabric api with qsl if I'm not mistaken, so automatically installing fabric api (and not allowing the user to remove it) doesn't work well in that case

**Comment by jakobkmar** [comment]
@sparrowofsong that's another issue, fabric-api is not an optional dependency in that case, but a required one - this issue is about optional dependencies not being installed at all at the moment
